modules/mod_ginger_spa/dispatch/dispatch

```diff
@@ -1,3 +1,6 @@
 [{home, [], controller_template, [ {template, "page.tpl"}, {id, home} ]},
- {search, ["search"], controller_template, [ {template, "page.tpl"} ]}
+ {search, ["search"], controller_template, [ {template, "page.tpl"} ]},
+
+ % Render a template the path is the template, pass arguments via the query post or get
+ {spa_render, ["api", "render-template", "public", '*' ], controller_ginger_spa_template, []}
```

Comment:
She earlier comment on public/test.tpl, maybe we should make the default spa_render rule specific, for discoverability of the default test template.